client/src/unifyfs-fixed.c

```diff
@@ -543,78 +695,64 @@ static int unifyfs_logio_chunk_write(
     /* get global file id for this file */
     int gfid = unifyfs_gfid_from_fid(fid);
 
-    /* define an new index entry for this write operation */
-    unifyfs_index_t cur_idx;
-    cur_idx.gfid     = gfid;
-    cur_idx.file_pos = pos;
-    cur_idx.log_pos  = log_offset;
-    cur_idx.length   = count;
+    /* Update our write metadata with the new write */
+    rc = unifyfs_logio_add_write_meta_to_index(meta, pos, log_offset,
```

Comment:
let's move the type definition of int rc here and drop it from the top of the function, since I don't think we use it anywhere else above.

Reply:
Ok